src/features/profile/DangerCard.js

```diff
@@ -10,15 +10,14 @@ import {
 } from "@material-ui/core"
 import { ErrorOutline as DangerIcon } from "@material-ui/icons"
 
+import { FATHOM_DELETE_ALL_ENTRIES, FATHOM_DELETE_ALL_ENTRIES_CONFIRMED, FATHOM_DELETE_ALL_MENSES_TAGS } from "../../constants";
```

Comment:
**nitpick (non-blocking):** Pull down the changes locally and give it a save so its formatted properly.

Reply:
I don't understand, so I will learn this some other time